File: LeetCode/852.cpp
//Link: https://leetcode.com/problems/peak-index-in-a-mountain-array/

class Solution {
public:
    int peakIndexInMountainArray(vector<int>& arr) {
        int index = 0;
        for(int i = 0 ; i < arr.size() ; i++)
            if(arr[i] > arr[index])
                index = i;
        return index;
    }
};

File: LeetCode/py/852.py
#Link: https://leetcode.com/problems/peak-index-in-a-mountain-array/

class Solution:
    def peakIndexInMountainArray(self, arr: List[int]) -> int:
        index = 0
        for i in range(len(arr)):
            if arr[i] > arr[index]:
                index = i
        return index
